packages/oui-numeric/src/numeric.html

```diff
@@ -4,13 +4,17 @@
           ng-disabled="$ctrl.disabled || $ctrl.model <= $ctrl.min"
           ng-click="$ctrl.decrement()"
           tabindex="-1"
-          aria-labelledby="-">
+          aria-hidden="true">
     <i class="oui-icon oui-icon-remove" aria-hidden="true"></i>
```

Comment:
(optionnal)

`aria-hidden` still necessary, if the parent has it ?

Reply:
you're right, i forgot there was an aria-hidden on <i>, will remove it